### PR TITLE
refactor: [M3-6356] - MUI v5 Migration - `Components > LineGraph`

### PR DESCRIPTION
**Pull request by jaalah-akamai**
## Description 📝
Refactor `LineGraph` with `styled` api

## Major Changes 🔄
- Removed `jss` and `makeStyles` for `styled` api
- Updated imports/exports
- Fixed bug for `LinodeEntityDetail.tsx` causing console error `Cannot have a div within a p tag`
- Uncovered bug for `NetworkGraphs.tsx` where the graphs weren't spanning full width after properly refactoring (probably was masked by some collision between `jss` and new styles`

## Preview 📷
| Before  | After   |
| ------- | ------- |
| Screenshot 2023-06-16 at 2 13 35 PM | Screenshot 2023-06-16 at 2 14 19 PM |

## How to test 🧪
1. Go to a linode detail page where you see line graphs
2. You shouldn't notice any observable changes with exception of what I fixed
3. Check different breakpoints
4. Toggle legends (buttons) and ensure they're crossed out

### CODE REVIEW

File: packages/manager/src/features/Linodes/LinodeEntityDetail.tsx [hunk 1]
```diff
@@ -820,21 +823,29 @@ export const Footer = React.memo((props: FooterProps) => {
...
+                Region:
+              </Box>{' '}
+              {linodeRegionDisplay}
             </Typography>
           )}
         </Box>
         <Box sx={sxBox}>
           <Typography sx={{ ...sxListItem, ...sxListItemFirstChild }}>
-            <Box sx={sxLabel}>Linode ID:</Box> {linodeId}
+            <Box component="span" sx={sxLabel}>
```
Comment (jaalah-akamai) [inline packages/manager/src/features/Linodes/LinodeEntityDetail.tsx:835]: Fixed bug for LinodeEntityDetail.tsx causing console error `Cannot have a div within a p tag`

File: packages/manager/src/features/Linodes/LinodesDetail/LinodeSummary/NetworkGraphs.tsx [hunk 2]
```diff
@@ -168,7 +168,7 @@ export const NetworkGraphs: React.FC<Props> = (props) => {
   };
 
   return (
-    <Grid container className={`${classes.graphGrids} p0`} spacing={4}>
+    <Grid container className={`${classes.graphGrids} p0`} xs={12} spacing={4}>
```
Comment (jaalah-akamai) [inline packages/manager/src/features/Linodes/LinodesDetail/LinodeSummary/NetworkGraphs.tsx:171]: This was missing, should match what we're doing here: https://github.com/linode/manager/blob/develop/packages/manager/src/features/Linodes/LinodesDetail/LinodeSummary/LinodeSummary.tsx#L293

File: packages/manager/src/components/LineGraph/LineGraph.tsx [hunk 3]
```diff
@@ -289,6 +283,21 @@ const LineGraph: React.FC<CombinedProps> = (props: CombinedProps) => {
...
+    ...sxTypography,
+    color: theme.textColors.tableHeader,
+  };
+
+  const sxLegend = {
+    [theme.breakpoints.up('md')]: {
+      width: '38%',
+    },
+  };
+
```
Comment (jaalah-akamai) [inline packages/manager/src/components/LineGraph/LineGraph.tsx:300]: Didn't think these warranted using `styled` api

**Review by bnussman-akamai — APPROVED** [review]
Nice! Thank you for fixing the Linodes Details page, it was driving me nuts 🌰

**Review by bnussman-akamai — CHANGES_REQUESTED** [review]
Testing BanksBot

**Review by bnussman-akamai — APPROVED** [review]

**Review by cpathipa — APPROVED** [review]

**Pull request merged**